srcs/elements/Player.cpp

```diff
@@ -187,35 +270,68 @@ bool	Player::takeBonus(BonusType::Enum bonus) {
 	return true;
 }
 
+/**
+ * @brief increment bomb, clamp to totalBombs
+ *
+ */
 void	Player::addBomb() {
 	bombs++;
 	if (bombs > totalBombs)
 		bombs = totalBombs;
 }
 
+/**
+ * @brief called on animation end if passed to Model
+ *
+ * @param animName the current animation name
+ */
+void	Player::animEndCb(std::string animName) {
+	// logDebug("animEndCb -> " << animName);
+	if (animName == "Armature|drop") {
+		_putBomb();
+		setStatus(EntityStatus::IDLE);
+	}
+	else if (animName == "Armature|death") {
+		logInfo("Player is dead.")
+		game.state = GameState::GAME_OVER;
+	}
+}
+
 // -- Private Methods ----------------------------------------------------------
 
-void	Player::_move(float const dTime) {
+void	Player::_move() {
+	bool		moved = false;
 	glm::vec3	dir = glm::vec3(0, front.y, 0);
 
 	if (Inputs::getKey(InputType::UP)) {
+		moved = true;
 		dir.z -= 1;
 	}
 	if (Inputs::getKey(InputType::RIGHT)) {
+		moved = true;
 		dir.x += 1;
 	}
 	if (Inputs::getKey(InputType::DOWN)) {
+		moved = true;
 		dir.z += 1;
 	}
 	if (Inputs::getKey(InputType::LEFT)) {
+		moved = true;
 		dir.x -= 1;
 	}
-	_moveTo(dir, dTime);
+	_moveTo(dir);
```

Comment:
```suggestion
	_moveTo(dir, 1);
```
Je te conseille de changer l'offset pour tourner automatiquement

Reply:
ah oui j'ai oublié de changer la taille de la colision du player aussi, ça sert à quoi le `, 1` ?

Reply:
en gros tu met un offset qui permet de dire a partir de quand tu commence a tourner automatiquement (si c'est 0 ca va jamais tourner automatiquement, 0.3 uniquement quand on est presque au bon endroit, ...)